docker/deployment/entrypoint.sh

```diff
@@ -6,15 +6,56 @@ if [ "$#" -gt 0 ]; then
     shift
 fi
 
+_octane() {
+  # exit on config update fail
+  set -e
+
+  /app/artisan config:cache
+  /app/artisan route:cache
+
+  /app/docker/deployment/entrypoint.sh config_reloader &
```

Comment:
I'm not sure the deployment version should be trying to watch and reload by default 🤔

Reply:
production probably shouldn't even use `.env` file to begin with...

(similarly for oauth key but notification server will need to be updated to be able to load from env variable as well)

Reply:
why not? i'm using it to enable faster reloads in production. i can apply envs to containers directly instead but that will require container restarts for any env change.

@peppy what are your thoughts on the matter?

Reply:
We don't change env often enough in production to require watching, I don't think. I would expect to have to restart the containers when changing env settings.

Keep it simple for now I guess (no watching required)